platform/posix/transport/utest/openssl_utest.c

```diff
@@ -768,12 +768,37 @@ void test_Openssl_Send_Network_Error( void )
 
     /* Several errors can be returned from #SSL_get_error as mentioned here:
      * https://www.openssl.org/docs/man1.1.1/man3/SSL_get_error.html */
-    SSL_get_error_ExpectAnyArgsAndReturn( SSL_ERROR_WANT_WRITE );
+    SSL_get_error_ExpectAnyArgsAndReturn( SSL_READ_WRITE_ERROR );
```

Comment:
I don't believe this is a value able to be returned by SSL_get_error(). Might want to use something like SSL_ERROR_ZERO_RETURN instead